pom.xml

```diff
@@ -596,6 +597,9 @@
 
               <!-- Due to the removal of deprecated methods -->
               <exclude>org.apache.parquet.arrow.schema.SchemaMapping</exclude>
+
+              <exclude>org.apache.parquet.io.api.Binary</exclude>
```

Comment:
Thanks for rebasing! Could you change the exclusion to the level of a specific method? Class level seems too wide to me.